src/elements/content-sidebar/SidebarFileProperties.js

```diff
@@ -55,7 +55,7 @@ const SidebarFileProperties = ({
                     : {}
             }
             size={getFileSize(file.size, intl.locale)}
-            uploader={getProp(file, 'created_by.name')}
+            uploader={getProp(file, 'uploader_display_name')}
```

Comment:
Is the `uploader_display_name` always populated even in non forms use cases? Or should it be a combination of `created_by.name` and `uploader_display_name`?

Reply:
ah, good catch... let me reword the logic

Reply:
@ConradJChan addressed